familyplanningmodule/src/main/res/drawable/save_bg.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+    <corners
+        android:topLeftRadius="4dp"
+        android:topRightRadius="4dp"
+        android:bottomRightRadius="4dp"
+        android:bottomLeftRadius="4dp">
+    </corners>
+    <stroke
+        android:width="1dp"
+        android:color="#0085C5">
+    </stroke>
+    <solid
+        android:color="#0085C5">
```

Comment:
Can you move these text-colors and text-color-hints to the `colors.xml` files and reuse them